_posts/2018-11-15-using-vault-with-strimzi.md

```diff
@@ -48,9 +48,9 @@ Now, in order to bring your own CA, you need to do this yourself and add the nec
 
 Using Vault to create your own CA arguably gives you more control over your TLS configuration.
 
-Consider a Strimzi Kafka cluster with a Kafka client you have developed. The cluster requires that clients authenticate themselves using SSL, i.e. `ssl.client.auth="required"`.
+Consider a Strimzi Kafka cluster with a Kafka client you have developed. The cluster has been configured with a TLS listener and as such requires that clients authenticate themselves using TLS.
```

Comment:
actually other than the `tls` listener you have to configure the `authentication` to TLS as well in order to enable TLS client authentication.

Reply:
@ppatierno Good point.